generic3g/specs/FieldSpec.F90

```diff
@@ -236,60 +236,13 @@ subroutine allocate(this, rc)
       call ESMF_FieldGet(this%payload, status=fstatus, _RC)
       _ASSERT(fstatus == ESMF_FIELDSTATUS_COMPLETE, 'ESMF field status problem.')
       if (allocated(this%default_value)) then
-         call set_field_default(_RC)
+         call FieldSet(this%payload, this%default_value, _RC)
```

Comment:
This looks weird to me.   I usually have camel case just for constructors and this is a subroutine call.

Reply:
Am guessing this comes from the "Darian" layer and I did not force camel case there when I should have ...  (will open separate ticket for that.)